src/blockchain_db/sqlite/db_sqlite.cpp

```diff
@@ -224,6 +224,23 @@ namespace cryptonote {
     return payments;
   }
 
+  uint64_t BlockchainSQLite::get_accrued_earnings(const std::string& address) {
+    LOG_PRINT_L3("BlockchainDB_SQLITE::" << __func__);
+
+    SQLite::Statement select_earnings {
+      db,
+      "SELECT amount FROM batched_payments_accrued WHERE address = ?;"
+    };
+    select_earnings.bind(1, address);
+
+    uint64_t amount{};
+    while (select_earnings.executeStep()) {
+      amount = static_cast<uint64_t>(select_earnings.getColumn(0).getInt64() / 1000);
```

Comment:
why does this divide by 1000?

Reply:
> why does this divide by 1000?

Because the database calculates and stores amounts in atomic units * 1000 to minimize integer math truncation errors.